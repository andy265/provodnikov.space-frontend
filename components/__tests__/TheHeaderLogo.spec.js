const { createRenderer } = require('vue-server-renderer')
const { shallow } = require('vue-test-utils')
const TheHeaderLogo = require('~/components/TheHeaderLogo')

describe('TheHeaderLogo.vue', () => {
  it('has same HTML structure', () => {
    let wrapper = shallow(TheHeaderLogo)
    let renderer = createRenderer()
    // eslint-disable-next-line
    renderer.renderToString(wrapper.vm, (err, str) => {
      expect(str).toMatchSnapshot()
    })
  })
})
