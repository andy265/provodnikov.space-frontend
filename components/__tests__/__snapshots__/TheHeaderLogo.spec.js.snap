// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheHeaderLogo.vue has same HTML structure 1`] = `
<div class="the-header-logo">
    <div class="the-header-logo__image"></div>
</div>
`;
